text/0028-indexing-csv.md

```diff
@@ -166,6 +171,7 @@ curl \
 - 🔴 Sending a payload excessing the limit will lead to a 413 Payload Too Large - **payload_too_large** error code.
 - 🔴 Sending an invalid CSV format will lead to a 400 bad_request - **malformed_payload** error code.
 - 🔴 Sending a CSV header that does not conform to the specification will lead to a 400 bad_request - **malformed_payload** error code.
+- 🔴 Sending an invalid csv delimiter: not exactly one ASCII char. This will lead to a 400 bad_request - **invalid_index_csv_delimiter** error code.
```

Comment:
```suggestion
- 🔴 Sending an invalid csv delimiter: not exactly one ASCII char. This will lead to a 400 bad_request - **invalid_document_csv_delimiter** error code.
```

Would'nt it be more clear to rename the error `invalid_document_csv_delimiter` since we are operating on the `/indexes/:indexUid/**documents**` resources and since the delimiter is applied to a file containing documents?

Reply:
Yep, we can do that